.evergreen/serverless/README.md

```diff
@@ -0,0 +1,51 @@
+# Serverless Scripts
+
+The scripts in this directory are used to test MongoDB Serverless instances on Atlas.
+
+## Prequisites
+
+Set up your environment to obtain secrets from the Drivers [AWS Vault](https://wiki.corp.mongodb.com/display/DRIVERS/Using+AWS+Secrets+Manager+to+Store+Testing+Secrets).
+
+## Usage
+
+First, get the appropriate secrets from the vault using:
+
+```bash
+bash $DRIVERS_TOOLS/.evergreen/serverless/setup-secrets.sh
+```
+
+If targeting the dev version of Serverless, use:
+
+```bash
+$DRIVERS_TOOLS/.evergreen/serverless/setup-secrets.sh serverless-next
+```
+
+The full task group might look like:
```

Comment:
What is this referencing? Feels very out of nowhere when compared to the previous lines.